closed/Intel/code/dlrm-99.9/pytorch-cpu/run_mlperf.sh

```diff
@@ -26,17 +26,18 @@ do
 done
 
 WORK_DIR=$PWD
-
-source ./setup_dataset.sh
+pwd
+ls
+source /workspace/setup_dataset.sh
 
 if [ ${mode} = "offline" ]; then
     scenario="offline"
-    source ./setup_env_offline.sh
+    source /workspace/setup_env_offline_ICX.sh
```

Comment:
could we remove ICX keyword since it also works on SPR?

Reply:
Change key word "ICX" to "AWS", because there is another "setup_env_offline.sh" script in this dir.